cmd/libs/go2idl/generator/execute.go

```diff
@@ -97,27 +89,27 @@ func (f *file) assemble(w io.Writer) {
 		fmt.Fprint(w, ")\n\n")
 	}
 
-	if f.vars.Len() > 0 {
+	if f.Vars.Len() > 0 {
 		fmt.Fprint(w, "var (\n")
-		w.Write(f.vars.Bytes())
+		w.Write(f.Vars.Bytes())
 		fmt.Fprint(w, ")\n\n")
 	}
 
-	if f.consts.Len() > 0 {
+	if f.Consts.Len() > 0 {
 		fmt.Fprint(w, "const (\n")
-		w.Write(f.consts.Bytes())
+		w.Write(f.Consts.Bytes())
 		fmt.Fprint(w, ")\n\n")
 	}
 
-	w.Write(f.body.Bytes())
+	w.Write(f.Body.Bytes())
 }
 
 // format should be one line only, and not end with \n.
 func addIndentHeaderComment(b *bytes.Buffer, format string, args ...interface{}) {
 	if b.Len() > 0 {
-		fmt.Fprintf(b, "\n\t// "+format+"\n", args...)
+		fmt.Fprintf(b, "\n// "+format+"\n", args...)
```

Comment:
nit: I'm not sure we need it - it will be added by gofmt anyway (note that the whole code is not formatted, to formatting just these comments makes it non-consistent).